frontend/src/pages/AdminPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { FunctionComponent } from 'react'
+import { Link } from 'react-router-dom'
+import LayoutWithNav from '../layouts/LayoutWithNav'
+
+/**
+ * The root page for all admin activities
+ */
+const AdminPage: FunctionComponent = () => {
+  return (
+    <LayoutWithNav>
+      <div className="text-da-navy-100 flex flex-col space-y-2">
+        <Link to="/groups">Groups</Link>
+        <Link to="/users">Users (in construction)</Link>
+      </div>
+    </LayoutWithNav>
+  )
+}
```

Comment:
Nothing fancy for now, it's just a way to navigate around the site with ease. Will iterate in the future!

<img width="1059" alt="Screen Shot 2021-03-07 at 10 59 44 AM" src="https://user-images.githubusercontent.com/3411183/110251187-3c340080-7f34-11eb-8a61-f90e5ed6a08c.png">